zest/releaser/utils.py

```diff
@@ -31,6 +49,42 @@ def loglevel():
     return logging.INFO
 
 
+def read_text_file(filename, encoding=None):
+    # Unless specified manually, We have no way of knowing what text
+    # encoding this file may be in.
+    # The 'open' method uses the default system encoding to read text
+    # files in Python 3 or falls back to utf-8.
```

Comment:
It's a shame that [tokenize.open](https://docs.python.org/3.4/library/tokenize.html#tokenize.open) and [tokenize.detect_encoding](https://docs.python.org/3.4/library/tokenize.html#tokenize.detect_encoding) never made it into Python 2.7.